src/renderer/domains/network/account/service.ts

```diff
@@ -34,9 +34,12 @@ function uniqId(account: AnyAccountDraft) {
 }
 
 function isCryptoMatch(account: Pick<AnyAccount, 'cryptoType'>, chain: Chain): boolean {
-  const cryptoType = networkUtils.isEthereumBased(chain.options) ? CryptoType.ETHEREUM : CryptoType.SR25519;
+  const isEthereumBased = networkUtils.isEthereumBased(chain.options);
+  if (isEthereumBased) {
+    return [CryptoType.ED25519, CryptoType.ECDSA, CryptoType.ETHEREUM].includes(account.cryptoType);
```

Comment:
Can you please explain why do check for ED25519 and ECDSA?

* There is no way ED25519 is valid for Ethereum crypto
* For ECDSA I imagine some signers mistekently return ECDSA instead of  ETHERUM for Evm-addressable Substrate chains?